Website/Src/Views/Shared/_Layout.cshtml

```diff
@@ -8,6 +7,16 @@
 @inject SignInManager<ApplicationUser> SignInManager
 @inject UserManager<ApplicationUser> UserManager
 
+@{
+    var userSettingsObject = UserSettingsProvider.Get(UserManager.GetUserId(User));
+    var themeLocalUrl = "~/lib/bootstrap/dist/css/bootstrap";
```

Comment:
Any reason not to specify `.css` here instead of concatting below?

Reply:
This because we support .min.css below